ansible/roles/proteus-db/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Create database schema files
+  copy:
+    src: "{{ item }}"
+    dest: "{{ proteus_db_schema_path }}"
+    mode: 0644
+  with_items: "{{ proteus_db_files }}"
+
+- name: Run database schema commands
+  shell: "psql -U {{ proteus_db_user}} -h 127.0.0.1 -d {{ proteus_db_name }} -f {{ proteus_db_schema_path}}/{{ item }}"
+  environment:
+    PGPASSWORD: "{{ proteus_db_password }}"
+  with_items: "{{ proteus_db_files }}"
```

Comment:
No, I think you were the one suggesting that as a solution, but the issue in the migration scripts was solved, so there is no plan to go that route in the near future.